qiskit_metal/analyses/quantization/lumped_capacitive.py

```diff
@@ -23,20 +23,24 @@
 import io
 import re
 from pathlib import Path
-from typing import List
+from typing import List, Union
 
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import scipy.optimize as opt
 from pint import UnitRegistry
 
+from pyEPR.ansys import ureg
+from pyEPR.calcs.convert import Convert
```

Comment:
The dependency on pyEPR is actually a limiter for generalization. Do you think there is a way to bring in those methods?

Reply:
Yeah I think there's a way. No need to import pyEPR. I will fix that

Reply:
Sounds like this piece here is the only one left, then we can approve the PR. If this one is too troublesome, we can postpone it to a later PR as well @quantum-quant . Up to you.

Reply:
I removed the import _ureg_ from pyEPR since we can import from pint directly. But the _Convert_ import is a little more complicated. This class is needed for unit conversions but it currently lives inside pyEPR. Certainly we don't want to rewrite it. This is more of a bigger architecture question now: do we want pyEPR to be the provider of basic math and physics libraries needed for physics analysis in metal. Perhaps we don't want to answer that question yet in this PR?

Reply:
Please bring this question up next Tuesday